### PR TITLE
Feature/structure of the website

### PR DESCRIPTION
**Pull request by josch87**
Created structure and first content of webpage. No styling yet.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+          <span>Atlassian Bamboo</span>
+          <span>MS Visual Studio Code</span>
+          <span>MS Office (incl. Access)</span>
+          <span>Adobe Photoshop</span>
+          <span>Adobe InDesign</span>
+          <span>WordPress</span>
+          <span>FirstSpirit</span>
+          <span>Figma</span>
+          <span>DBeaver</span>
+          <span>AGS Server Studio</span>
```
Comment (vaupunkt) [inline index.html:53]: you could do <ul> instead of <span>

File: index.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      <section id="contact">
+        <div>
+          Call me:
+          <a href="tel:+49123456789" title="Click 2 call">0123 456789</a> |
+          Write me:
+          <a
+            href="mailto:mail@schoenefische.de?subject=Nice%20Webpage!&body=Hello%2C%0D%0A%0D%0Amy%20name%20is%20%24%7BFirstname%7D%20and%20I%20like%20your%20webpage.%20Can%20I%20hire%20you%20as%20developer%20to%20get%20my%20stuff%20done%3F%0D%0A%0D%0A%0D%0ABest%20regards"
+            title="Click 2 write"
+            >mail@schoenefische.de</a
+          >
```
Comment (vaupunkt) [inline index.html:65]: does this break in the end-tag work?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+          <a href="https://www.xing.com/profile/Aljoscha_Zoeller/cv"
+            ><img src="assets/logos/xing-logo.png"
+          /></a>
+          <figcaption>
+            <a href="https://www.xing.com/profile/Aljoscha_Zoeller/cv">Xing</a>
+          </figcaption>
+        </figure>
+      </section>
+      <section id="about-me">
+        <div id="bio">
```
Comment (vaupunkt) [inline index.html:93]: maybe you could use `<article>` for these divs?

**Review by vaupunkt — APPROVED** [review]
looks great, besides of some small comments 👍

File: index.html (hunk 1 above)
Comment (josch87) [reply index.html:53]: I want to style the items to badges instead of a list, thats why I use span. Thanks for the hint.

File: index.html (hunk 2 above)
Comment (josch87) [reply index.html:65]: Unfortunately the prettier plugin is pushing this into the next line when saving the file. Doesn't look good in the code, I know...

File: index.html (hunk 3 above)
Comment (josch87) [reply index.html:93]: Thx, I'll change it to article.

**Pull request merged**